src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -5,11 +5,27 @@
  * Do not remove no-args constructor
  */
 public class Bulldozer extends Machine {
+    private double bladeWidth;
+    private int enginePower;
```

Comment:
Don't forget about access modifiers for your custom fields in `Bulldozer` class. Consider using private access modifier to follow encapsulation principle.